docs/installation.zh.md

```diff
@@ -99,12 +153,27 @@ kubectl port-forward -n crane-system svc/craned 9090
 
 通过 YAML 安装 `Crane` 。
 
-```console
-git checkout v0.2.0
-kubectl apply -f deploy/manifests 
-kubectl apply -f deploy/craned 
-kubectl apply -f deploy/metric-adapter
-```
+=== "Main"
+
+    ```bash
+    git clone https://github.com/gocrane/crane.git
+    CRANE_LATEST_VERSION=$(curl -s https://api.github.com/repos/gocrane/crane/releases/latest | grep -oP '"tag_name": "\K(.*)(?=")')
+    git checkout $CRANE_LATEST_VERSION
+    kubectl apply -f deploy/manifests 
+    kubectl apply -f deploy/craned 
+    kubectl apply -f deploy/metric-adapter
+    ```
+
+=== "Mirror"
+
```

Comment:
It will be cleaner  to  introduce mirror in a separate section, you can explain how to change a cmd into a used mirror cmd by an example. We don't need to explain it for every cmd